spring-boot-project/spring-boot-starters/spring-boot-starter-parent/build.gradle

```diff
@@ -191,7 +191,7 @@ publishing.publications.withType(MavenPublication) {
 								dependency {
 									delegate.groupId('org.springframework.boot')
 									delegate.artifactId('spring-boot-maven-plugin')
-									delegate.version('${project.version}')
+									delegate.version('${version}')
```

Comment:
`gradle publishToMavenLocal` generates the following for me: 

```
<dependencies>
            <dependency>
              <groupId>org.springframework.boot</groupId>
              <artifactId>spring-boot-maven-plugin</artifactId>
              <version>${version}</version>
            </dependency>
          </dependencies>
```

I believe it should generate the following:

```
<dependencies>
            <dependency>
              <groupId>org.springframework.boot</groupId>
              <artifactId>spring-boot-maven-plugin</artifactId>
              <version>2.3.0.BUILD-SNAPSHOT</version>
            </dependency>
          </dependencies>
```

Reply:
The following will work:

```groovy
delegate.version("${project.version}")
```

This aligns with how the parent version is configured earlier in the same `build.gradle` file:

```groovy
parent {
	delegate.groupId("${project.group}")
	delegate.artifactId("spring-boot-dependencies")
	delegate.version("${project.version}")
}
```

The change can be verified with the following command:

```
./gradlew spring-boot-project:spring-boot-starters:spring-boot-starter-parent:generatePom
```

This will write the generated pom to `spring-boot-project/spring-boot-starters/spring-boot-starter-parent/build/publications/maven/pom-default.xml`

Reply:
@wilkinsona  If I'm understanding you correctly does that mean this PR is simply not needed?

Reply:
No, I think it's still needed, but in a modified form. We currently have `'${project.version}'`. We need `"${project.version}"` instead. Note the use of `"` rather than `'`.